### PR TITLE
Fix menu layout

### PR DESCRIPTION
**Pull request by fabienrohrer**
- [x] Simplified menu layout
- [x] Removed the menu scroll bar (by fixing the layout)
- [x] Replaced the navigation text by icons

### CODE REVIEW

**Comment by fabienrohrer** [comment]
I improved the navigation shrink issue by replacing the navigation text elements by icons + tooltips.

It's still not perfect, but a step further, I think.

capture d ecran 2016-04-21 a 15 35 20
capture d ecran 2016-04-21 a 15 35 27

**Comment by fabienrohrer** [comment]
👍

File: index.html [hunk 1]
```diff
@@ -9,16 +9,18 @@
...
-        <tr>
-          <td><a id="previous">&lt; Previous</a></td>
-          <td><a id="up">^ Up</a></td>
-          <td><a id="next">Next &gt;</a></td>
-        </tr>
-      </table>
+      <div id="navigation">
+        <table>
+          <tr>
+            <td colspan="3"><a id="toc" title="Table of Contents">&#x25B3;</a></td>
```
Comment (omichel) [inline index.html]: What about using this symbol instead: <b>&equiv;</b> `<b>&equiv;</b>` for the table of contents?

Comment (fabienrohrer) [inline index.html]: Indeed, it would be much better :-)
I'm changing this, and post a new screenshot.

**Comment by fabienrohrer** [comment]
Ok, here is the new result. Much better indeed.
I also increased the menu font size.

capture d ecran 2016-04-21 a 17 19 45

**Comment by fabienrohrer** [comment]
👍

**Comment by omichel** [comment]
:+1:

**Pull request merged**